ha_cluster_exporter.go

```diff
@@ -282,11 +282,11 @@ func main() {
 				started = true
 			}
 
-			log.Println("[INFO]: Reading quorum status...")
+			log.Infoln("Reading quorum status...")
```

Comment:
ditto. All this info thing it might be only debug. I think for moment we can just fix it later